app/models/game.rb

```diff
@@ -37,4 +42,9 @@ def initialize_board
     
   end
   
+  # For testing
+  def clear_board_pieces
```

Comment:
maybe a better name would be clear_board! - 

1. it clears the board
2. !-notation indicates (only per convention, not programmatically) that something is getting changed in the database..